tests/driver_sht1x/Makefile

```diff
@@ -0,0 +1,6 @@
+BOARD ?= native
+include ../Makefile.tests_common
+
+USEMODULE += sht1x
```

Comment:
Same comment about the version of sensor, better use the pseudomodule which will implicitly load the common concrete module.

Reply:
Sure the device specific pseudo modules should be used for unit tests that are not device specific? I would prefer to keep sht1x here, otherwise it is not obvious that is applies to all variants.